lms/djangoapps/branding/api.py

```diff
@@ -0,0 +1,135 @@
+"""Edx footer API
+"""
+import logging
+from util.json_request import JsonResponse
+from django.conf import settings
+from django.utils.translation import ugettext as _
+from microsite_configuration import microsite
+
+log = logging.getLogger("edx.footer")
+
+
+def get_footer():
+    """ Get the footer links json
+
+    Returns:
+        Dict of footer links
+    """
+    site_name = microsite.get_value('SITE_NAME', settings.SITE_NAME)
+    context = dict()
+    context["copy_right"] = copy_right()
+    context["heading"] = heading()
+    context["logo_img"] = "{site_name}/sites/all/themes/atedx/images/edx-logo-footer.png".format(site_name=site_name)
```

Comment:
Don't we need to use `static.url()` to use the cache-busting version of the URL?